README.md

```diff
@@ -40,6 +40,19 @@ image:
 Additionally for the best user experience and optimized `sti` operation we suggest images
 to have `/bin/sh` and `tar` commands available.
 
+Pruning the contents of the source tree is possible if the user supplies a
+`.s2iignore` file in the root directory of the source repository.  Supply line by line the regular
+expression that best captures a set of files and directories you want removed.  You can also cite
```

Comment:
I'm in a different team now, I can be picky, besides I'm too far away to get my ass kicked for doing so :stuck_out_tongue_winking_eye: